tnp/templates/consent/edit_profile.html

```diff
@@ -0,0 +1,364 @@
+{% extends 'base.html' %}
```

Comment:
Thinking of moving this page to a card based UI as seen in `job.html` file. Tell me about your opinion? @abhishek981996